GOVERNANCE.md

```diff
@@ -1,114 +1,287 @@
+  to the Code of Conduct Committee.
+
+Additionally, the project's Approvers Committee will select, by majority
+vote, one representative to the Steering Committee. This representative
+need not be a member of the subproject's Approvers Committee, and will
+be replaced or renewed by the committee annually.
+
+Should a member of the Approvers Committee cease being active in the subproject,
+violate the Code Of Conduct, or need to be removed for some other reason, they
+may be removed by a 2/3 majority vote of the other Committee members, or a
+majority vote of the Steering Committee.
+
+### Default Subproject Governance
+
+Subprojects without governing documents default to:
+
+#### Subproject Approvers Committee Responsibilities
+
+All active Approvers of the project, as defined in the Operator
+Framework Contributor Ladder, are members of the Maintainer Committee,
+which governs that project. The project's Approver Committee is
+responsible for the following project governance activities:
+
+* Ensuring that the project creates and publishes regular releases;
+* Holding regular, project-wide discussions on issues and planning for the project;
+* Monthly review of project contributors for advancement on the Contributor Ladder;
+* Making final decisions on project changes that involve controversial trade-offs;
+* Responding to security compromise reports;
+* Supporting the Code of Conduct within their project and referring violations to the Code of Conduct Committee.
+
+Should a member of the Approver Committee cease being active in the
+project, violate the Code Of Conduct, or need to be removed for some
+other reason, they may be removed by a 2/3 majority vote of the other
+Committee members, or a vote of the Steering Committee.
+
+
+#### Subproject Steering Committee Representative
+
+Additionally, the Approver Committee will select, by an election
+conducted by the Operator Framework Election officials, one
+representative of the project to the Operator Framework Steering
+Committee. This representative need not be a member of the project's
+Maintainer Committee, and will be replaced or renewed by the Committee
+annually.
+
+#### Subproject Governance Ammendments
+
+If the project members wish to update or supplement governance
+documents, this will require a ⅔ majority  of project Maintainers
+followed by a ⅔ majority approval from th e Operator Framework steering
+committee.                                
+                                          
+## Steering Committee                     
+                                          
+The overall Operator Framework umbrella project is governed by a Steering
+Committee, which is selected as follows:
+
+* One Maintainer representative from each member subproject
+* `7 - number_of_subprojects` General "Community Representatives", with a minimum of 3 or 4, whichever will result in an odd number of committee members.
+
+### Steering Committee Duties
```

Comment:
We decided to limit CNCF Maintainer status to just the Steering committee, right? Should have a blurb in here about representing OF to CNCF and related things like Kubecon

Reply:
added in latest commit